Patches/Forsakens/Forsakens_CE_Patch_ThingDefs_Weapons_BlackHydra.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Forsakens</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+			
+				<!-- NOTE: Black Hydra is currently disabled for CE, as the original mod incorrectly uses a thingDef rather than ThingDef, which is incompatible with CE patching -->
+				
+				<li Class="PatchOperationRemove">
+					<xpath>Defs/thingDef[defName="ForsakenHydra"]</xpath>
+				</li>
+
+				<!-- ========== Forsaken Black Hydra ========== -->
+
+				<!-- (CE implementation based on BFG 9000 from CP DOOM mod, with infinite ammo as per item description in original Forsakens mod) -->
+
+				<!-- TEMPORARILY DISABLED 
```

Comment:
What's the plan with this? As my concern is that if the patch gets merged, it'll get forgotten about.

Reply:
My plan is to leave the Black Hydra disabled until @Kik0hi fixes the `thingDef` / `ThingDef` issue on his end, as per his latest response on Steam Workshop. I'm checking the Forsakens mod page every day for any updates on his end:

![78996118-7eaf5a80-7b76-11ea-9bd3-277803f175b4](https://user-images.githubusercontent.com/4993380/79630913-d60d8600-8187-11ea-8a08-28d3910bd9d2.png)

In the extremely unlikely event that I forget about this, the absence of the Black Hydra doesn't adversely affect faction pawn generation and the general gameplay experience - there are plenty of other Forsakens rifles for pawns to choose from.

Reply:
Ah, yes! I remember you telling me that now, thank you for the reminder.